src/app/mork/factories/users.py

```diff
@@ -48,7 +48,11 @@ def service_statuses(
         if not create:
             return
 
-        service_states = {service: DeletionStatus.TO_DELETE for service in ServiceName}
+        service_states = {
+            service: DeletionStatus.TO_DELETE
+            for service in ServiceName
+            if service != ServiceName.BREVO
+        }
```

Comment:
What do you think to create a `DeprecatedServiceName` containing the `BREVO` key ? Then the choices for `user_service_statuses.service_name` column should a merge of both `DeprecatedServiceName` and `ServiceName`?

I feel [here](https://github.com/openfun/mork/blob/5a6a6bd65c210470bf86d2ef14f0b258b6176043/src/app/mork/celery/tasks/deletion.py#L129-L140) we do not want include Brevo anymore.

Reply:
Oh good catch! I like the idea of having a separate `DeprecatedServiceName`

Reply:
I didn't find a good way to have those two enums merged without being too hacky...
I'll stick to excluding the Brevo field on each insert